marketpulse/main/views.py

```diff
@@ -66,7 +74,7 @@ def edit_fxosprice(request, username='', id=None):
         messages.success(request, 'Contribution successfully saved')
         contribution_form = forms.ContributionForm()
         location_form = forms.LocationForm()
-        plan_formset = forms.PlanFormset()
+        plan_formset = PlanFormset()
 
         return redirect(reverse('main:activities'))
```

Comment:
I would suggest to redirect to the page listing either all or user activities instead of the main. This will make probably easier to edit multiple entries.